test/model/message_list_test.dart

```diff
@@ -221,5 +221,134 @@ void main() async {
         ..content.equals(newContent)
         ..lastEditTimestamp.equals(oldTimestamp);
     });
+
+    group('ReactionEvent handling', () {
+      test('add reaction', () async {
+        final store = await setupStore(stream);
+        final originalMessage = eg.streamMessage(id: 243, stream: stream, reactions: []);
+        final messageList = await messageListViewWithMessages([originalMessage], store, narrow);
+
+        final event = ReactionEvent(
+          id: 1,
+          op: ReactionOp.add,
+          emojiName: eg.unicodeEmojiReaction.emojiName,
+          emojiCode: eg.unicodeEmojiReaction.emojiCode,
```

Comment:
One way to make the tests more concise would be to make a local helper function to create these ReactionEvent objects, which could take a Reaction as a single argument. (Plus can skip the `id` parameter.)